llama/llama.go

```diff
@@ -205,17 +223,17 @@ func (m *Model) ShouldAddBOSToken() bool {
 	}
 }
 
-func (m *Model) ApplyLoraFromFile(loraPath string, scale float32, baseModelPath string, threads int) error {
+func (m *Model) ApplyLoraFromFile(context *Context, loraPath string, scale float32, threads int) error {
```

Comment:
Did we want to remove the lora patch here or in a future change?

Reply:
Not totally sure I understand - loraPath is still needed in this version, it's just moved in the argument list. On the other hand, baseModelPath is no longer supported by the new versions of the functions and has been removed from the argument list.

Reply:
Referring to this patch: https://github.com/ollama/ollama/blob/jmorganca/llama/llama/patches/09-lora.diff

Reply:
Ah, sorry, I thought "patch" was a typo for "path" given the quoted context.

My plan was to leave the patch for now and just not support it/use it in the Go server path. Once we remove server.cpp then we can remove the patch as well. Just trying to minimize collateral changes in the alternate server as we transition.

Reply:
That's a good point - I updated this commit to remove the LoRA patch from llama/patches